docs/software/How-to-install-Irishub.md

```diff
@@ -1,25 +1,31 @@
 # How to install IRIShub 
 
+### The Latest version of IRIShub : v0.13.1
+refer to : https://github.com/irisnet/irishub/releases/latest
+```
+Please replace <latest_iris_version> with v0.13.1 while using "git checkout" 
+```
+
 There are two ways to get `iris` running on your server. You can download the binary files from our release page, or you can download the source code and compile it locally.
 
 #### Download Binary Directly
 
 Go to the download page: 
 
-https://github.com/irisnet/irishub/releases/  
+https://github.com/irisnet/irishub/releases/latest
 
-then get the release v0.13.1 on your computer.
+then get the latest release on your computer.
 
 `unzip -C /usr/local/bin  iris$VERSION.$OS-$ARCH.zip` 
```

Comment:
Starting from v0.13.1, releases will not include pre-built binaries; users are required to build binaries from the source.

Reply:
ok，removed "Download Binary Directly"